src/commands/build-and-push-image.yml

```diff
@@ -18,6 +18,13 @@ parameters:
       Setup and use CircleCI's remote Docker environment for Docker and
       docker-compose commands? Not required if using the default executor
 
+  remote-docker-dlc:
```

Comment:
I'm on the fence about this naming. Maybe `remote-docker-caching` is clearer?

Reply:
The intent was to match [docker-orb](https://github.com/CircleCI-Public/docker-orb/pull/44/files) but per your recommendation, I went ahead and changed it to `remote-docker-layer-caching`